.github/workflows/preview.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           npm_config_arch: ${{ matrix.npm_config_arch }}
           ls_target: ${{ matrix.ls_target }}
       - name: Package VSIX
-        run: npm run package -- --pre-release --target=${{ matrix.vsce_target }}
+        run: npm run package -- --githubBranch f-tfc --pre-release --target=${{ matrix.vsce_target }}
```

Comment:
Apologies for early comment, I understand this is a draft. 😄 

I'm assuming you're planning to make this an input and also use it above in `actions/checkout` to actually checkout the branch, right? 👀

Reply:
Yep. WIP